cpp/oneapi/dal/algo/svm/detail/kernel_function.cpp

```diff
@@ -141,6 +145,41 @@ class daal_interop_rbf_kernel_impl : public kernel_function_impl {
     double sigma_;
 };
 
+template <typename Float, typename Method>
+class daal_interop_sigmoid_kernel_impl : public kernel_function_impl {
+public:
+    daal_interop_sigmoid_kernel_impl(double scale, double shift) : scale_(scale), shift_(shift) {}
+
+    daal_kf_t get_daal_kernel_function(bool is_dense) override {
+        if (is_dense) {
+            constexpr daal_sigmoid_kernel::Method daal_method = get_daal_dense_method();
+            auto alg = new daal_sigmoid_kernel::Batch<Float, daal_method>;
+            alg->parameter.scale = scale_;
+            alg->parameter.shift = shift_;
```

Comment:
in this case it will use a polynomial kernel! to use the sigmoidal one you must specify the kernel type:
```c++
alg->parameter.kernelType = KernelType::sigmoid
```